### DISCUSSION TITLE
Ynet Live [IL]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#399] [Show and tell]
### Channel ID

YnetLive.il

### Stream URL

https://hls-video-ynet.ynethd.com/0323/7a92522310b049209122e2ffbd920508/master.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://hls-video-ynet.ynethd.com/0323/7a92522310b049209122e2ffbd920508/master.m3u8
<img width="1411" alt="image" src="https://user-images.githubusercontent.com/7253922/232258948-ae9e5f27-b881-48ee-bf48-3cd12729b013.png">

**Comment by adinaveh** [comment]
This is vod 5min loop
